méli-mélo/th-canadaday/style.css

```diff
@@ -1,33 +1,59 @@
 /* Thematic red */
 .bg-canadaday,
 body .bg-canadaday {
-    background-color: #990000;
+    background-color: #A30D26;
 }
 
 .panel-canadaday,
 hr.brdr-canadaday {
-    border-color:#990000;
+    border-color: #A30D26;
 }
 
 /* Hover and focus state */
 a.bg-canadaday:hover,
 a.bg-canadaday:focus,
 button.bg-canadaday:hover,
 button.bg-canadaday:focus {
-    background-color: #6b0303;
+    background-color: #800613;
 }
 
 /* Active state */
 a.bg-canadaday:active,
 a.bg-canadaday:active:focus,
 a.bg-canadaday:active:hover,
 button.bg-canadaday:active {
-    background-color: #000000;
+    background-color: #680106 !important;
 }
 
 /* Force link text to remain white for hover, focus and active states. */
 a.bg-canadaday:hover,
 a.bg-canadaday:focus,
 a.bg-canadaday:active {
-    color: #fff;
+    color: #fff !important;
+}
+    
+/* Thematic beige */
+.bg-canadaday-beige,
+body .bg-canadaday-beige {
```

Comment:
Need a demo or be removed.

With gcweb-jekyll I do think you can use the front-matter "pageclass" like our pink-day working example demo